pubspec.yaml

```diff
@@ -20,8 +21,8 @@ dev_dependencies:
   build_runner: ^1.0.0
   objectbox_generator:
     path: generator
-  pedantic: ^1.10.0-nullsafety.0
-  test: ^1.16.0-nullsafety.0
+  pedantic: ^1.9.0
+  test: ^1.0.0
```

Comment:
I guess this is unrelated? But fine.

Reply:
nope, that's to restore compatibility with anything prior to 2.10